diff --git a/thirdparty_builtin/googletest-master-2020-01-07/googletest/include/gtest/internal/gtest-port.h b/thirdparty_builtin/googletest-master-2020-01-07/googletest/include/gtest/internal/gtest-port.h
index 0543da5..185fc88 100644
--- a/thirdparty_builtin/googletest-master-2020-01-07/googletest/include/gtest/internal/gtest-port.h
+++ b/thirdparty_builtin/googletest-master-2020-01-07/googletest/include/gtest/internal/gtest-port.h
@@ -581,6 +581,7 @@ typedef struct _RTL_CRITICAL_SECTION GTEST_CRITICAL_SECTION;
 # endif  // !GTEST_OS_WINDOWS_MOBILE
 #endif  // GTEST_HAS_STREAM_REDIRECTION
 
+#ifndef GTEST_HAS_DEATH_TEST
 // Determines whether to support death tests.
 // pops up a dialog window that cannot be suppressed programmatically.
 #if (GTEST_OS_LINUX || GTEST_OS_CYGWIN || GTEST_OS_SOLARIS ||             \
@@ -591,6 +592,7 @@ typedef struct _RTL_CRITICAL_SECTION GTEST_CRITICAL_SECTION;
      GTEST_OS_DRAGONFLY || GTEST_OS_GNU_KFREEBSD || GTEST_OS_HAIKU)
 # define GTEST_HAS_DEATH_TEST 1
 #endif
+#endif
 
 // Determines whether to support type-driven tests.
 
